images/linux/toolsets/toolset-1604.json

```diff
@@ -93,7 +93,8 @@
             "cmake;3.10.2.4988404",
             "patcher;v4",
             "ndk-bundle",
-            "platform-tools"
+            "platform-tools",
+            "cmdline-tools;latest"
```

Comment:
I think we don't need `latest` here, we don't have it for `ndk-bundle` but we still install the latest one, right?

Reply:
I think It is expected. In output of `sdkmanager --list`, It prints ndk-bundle as `ndk-bundle` and cmdline-tools as `cmdline-tools;latest`. Otherwise, it won't find it

Reply:
ok